tools/generate_svg_images.js

```diff
@@ -52,14 +20,12 @@ VF.Test.RUN_CANVAS_TESTS = false;
 VF.Test.RUN_SVG_TESTS = false;
 VF.Test.RUN_RAPHAEL_TESTS = false;
 VF.Test.RUN_NODE_TESTS = true;
-VF.Test.NODE_IMAGEDIR = path.resolve(__dirname, '..', 'build', 'images');
+VF.Test.NODE_IMAGEDIR = './build/images';
 
 // Create the image directory if it doesn't exist.
-try {
-  fs.accessSync(VF.Test.NODE_IMAGEDIR, fs.R_OK | fs.W_OK);
-} catch(e) {
-  fs.mkdirSync(VF.Test.NODE_IMAGEDIR);
-}
+fs.makeDirectory(VF.Test.NODE_IMAGEDIR);
```

Comment:
Yes it is, I had to update because SlimerJS filesystem api is different from Node.js. http://phantomjs.org/api/fs/